scripts/find_files.py

```diff
@@ -0,0 +1,35 @@
+"""
+ Copyright (C) 2022 Intel Corporation
+
+ SPDX-License-Identifier: MIT
+
+"""
+
+import os
+import sys
+import re
+
+"""
+    helper for obtaining all of the h/cpp files recursively,
+    starting from root_dir, with the exclusion of the build_dir
+"""
+
+def find_cpp_h_files_rec(root_dir, build_dir):
+    for path, dirs, files in os.walk(root_dir):
+        if build_dir in path:
+            continue
+        for file in files:
+            match = re.search("\.(h|cpp)$", file)
+            if match:
+                file_path = os.path.join(path, file)
+                print(file_path)
+
+def main():
+    if len(sys.argv) < 3:
```

Comment:
`if not exactly two args` is the condition you should check. You might consider using `argparse` for having nice output when using this script with `-h/--help` flags or in case of a wrong number of arguments passed. Or even wrong type of arguments.